app/src/main/res/layout/item_movie.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/colorPrimary">
+
+    <ImageView
+        android:id="@+id/iv"
```

Comment:
chorrada gigante: quizás a los identificadores deberíamos ponerle un nombre más explícito? En este caso, por ejemplo, podría ser "iv_grid_movie"

Reply:
No es ninguna chorrada. **Es muy buena pregunta, @adrianlazaro8 **. La respuesta es un poco larga. Trato de resumir:
Yo siempre trato de seguir las convenciones más estándar y buscar la practicidad de las cosas para ser más productivo en el día a día... dicho esto, he llegado a la conclusión de que hoy por hoy:
- Los nombres de ids de vistas deben ir en `lowerCamelCase`
- Deben ser lo más cortos posibles y no darles un nombre que sea redundante con respecto a contenedor que lo posee.

la segunda idea la corroboré con este charla de Kevlin Henney (si no la habéis visto, os la recomiendo encarecidamente. Aunque sea larga, merece la pena):
https://www.youtube.com/watch?v=ZsHMHukIlJY

Por tanto, si dentro de un layout, sólo hay un ImageView, éste debe de llamarse `iv`. Lo mismo para un TextView, `tv` y así sucesivamente.

Si tuviéramos dos TextViews, por ejemplo, podríamos llamarles algo así: `tvTitle` y  `tvContent`.

Reply:
Por cierto, se me olvidaba decir, que este modo facilita muchísimo las cosas para reutilizar código.

Reply:
¡Oído cocina Gabi! Está más que justificado :)

Muchas gracias por el enlace a la charla, que no la había visto